scripts/migrations.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+# The aim of this script is to generate missing initial migrations for apps
+# distributed with fun-apps.
+#
+# usage: scripts/migrations.sh [edxapp image tag]
+#
+# default: dogwood.3-1.0.0
+# example: scripts/migrations.sh eucalyptus.3-1.0.0
+
+declare EDXAPP_TAG
+declare PROJECT_DIRECTORY
+declare SETTINGS
+
+EDXAPP_TAG="${1:-dogwood.3-1.0.0}"
+PROJECT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." || exit; pwd -P)"
+
+# Terminal colors
+declare -r COLOR_INFO='\033[0;36m'
+declare -r COLOR_RESET='\033[0m'
+declare -r COLOR_WARNING='\033[0;33m'
+
+
+function usage(){
+
+  echo -e "usage: migrations.sh [edxapp image]\\n"
+  exit 10
+}
+
+
+function get_apps(){
+
+  find "${PROJECT_DIRECTORY}" -maxdepth 2 -name "models.py" -exec dirname {} \; | sort
+}
+
+
+function generate_app_tmp_settings(){
+
+  declare apps
+  declare tmp_dir
+  declare tmp_settings
+
+  apps=$(get_apps | grep -v "fun_api" | xargs -n 1 -I %% sh -c "basename %%" | sed "s/\\(.*\\)/  '\\1',/g")
```

Comment:
can you add a comment about what you are doing here ? I'm lost...

Reply:
Is it better now?

Reply:
See aff817b00